src/frontend/citizen-portal/src/app/components/scan-ticket/scan-ticket.component.ts

```diff
@@ -38,6 +34,11 @@ export class ScanTicketComponent implements OnInit {
       return;
     }
 
+    // get count summary info
+    this.ticket.counts.forEach(count => {
+      this.violationTicketCounts[count.count_no - 1] = count;
+    });
```

Comment:
```
this.ticket.counts.forEach(count => {
    this.violationTicketCounts.push(count);
});
```

If you would like to sort it, just add `.sort((a,b)=> a.count_no - b.count_no) `

Reply:
done